src/GregClient/Requests/GetUserPackages.cs

```diff
@@ -0,0 +1,46 @@
+﻿using RestSharp;
+
+namespace Greg.Requests
+{
+    /// <summary>
+    /// Request for getting latest versions of all packages published by the current user.
+    /// </summary>
+    public class GetUserPackages : PackageReferenceRequest
```

Comment:
Are there other `User` routes that don't actually take a user object? What do you think about calling this GetMyPackages?